### ISSUE TITLE
Add develop tag to docker hub image

### ISSUE THREAD

**Issue by yaron-idan** [Netflix/hubcommander#84]
Right now docker hub only allows pulling releases of hubcommander.
Could be very handy to be able to pull the develop branch directly, instead of waiting for a new release.
Right now if I want to deploy the develop branch I have to build the image myself, which is pretty easy thanks to the great build-docker.sh script, but having develop available for me directly from docker hub would be nice.
Thanks,
Yaron.

**Comment by mikegrima** [comment]
Agree -- this would be nice to have.

**Comment by yaron-idan** [comment]
Hey @mikegrima, could I please get feedback for the PR I've opened (#85)?
Thanks.

**Comment by mikegrima** [comment]
Merged it in. Thank you for the PR!

I think the last main portion is to modify: https://github.com/Netflix/hubcommander/blob/develop/publish_via_travis.sh#L36-L49 this so that we have a tag for stable/release, and one for unstable/development.

**Comment by yaron-idan** [comment]
True, I was in the mindset of creating an automated build in docker hub but your solution fits better with the current CI, I'll open another PR for it soon.
